oleesir/diaryReload
# feat(notes): implement notes-template

#### What does this PR do?
This pull request adds a notes template to the app

#### Description of Task to be completed?
* notes-template

#### How should this be manually tested?
* Navigate to the UI folder and open notes.html in your browser
* You will see a notes template

#### What are the relevant pivotal tracker stories?
 [#160895893](https://www.pivotaltracker.com/story/show/160895893)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-01 at 7 49 09 pm" src="https://user-images.githubusercontent.com/17679511/46309064-355e7200-c5b3-11e8-9b50-57256b7e0f8d.png">
